teps/0085-per-namespace-controller-configuration.md

```diff
@@ -79,6 +80,42 @@ As a contributor, I need to test my behavioral changes to ensure that they work
 - Operator can allow for configuration to be defined on per-namespace basis
 - User can specify and use a customized configuration for a given namespace
 
+## Proposal
+
+To enable the `tekton-pipelines-controller`'s feature flags configuration per namespace, we propose adding a new `configmap` with the same name `feature-flags` in each of the candidate namespaces, which contains the properties to override for all the `Pipelineruns` created in this specific namespace.
```

Comment:
please add a [link](https://github.com/tektoncd/pipeline/blob/main/docs/install.md#customizing-the-pipelines-controller-behavior) to the supported properties in the proposal.

Also, please update the documentation in the implementation PR to include this new `configmap` once the proposal is merged.

Reply:
Since the customization is spreading across the entire namespace, I am guessing all the resources under that namespace will be impacted, including `pipelineRun`, `taskRun`, and `run`.

Reply:
I would like for us to use a prefix here, like `tekton-feature-flags` (similar, `tekton-defaults`, …). Mainly to make sure we don't run the risk of conflictings with something else in the cluster/namespace that would have a different schema and mean something different.